hack/OWNERS

```diff
@@ -35,3 +35,4 @@ approvers:
   - soltysh # for sig-cli related stuff
   - dims # for local-up-cluster related files
   - spiffxp
+  - xichengliudui
```

Comment:
You should start in reviewers before approvers https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer